public/redirect.html

```diff
@@ -3,19 +3,36 @@
   <head></head>
   <body>
     <!--
-    This page is loaded after the authorize page done it's work, and this
-    simply updates local storage and the rest of torii takes over from There -->
+    This page is loaded after the authorize page has done it's work. This
+    simply updates local storage and the rest of torii takes over from there -->
+
+    <!-- load arcgis-rest-js -->
+    <script src="http://localhost:4200/assets/vendor.js"></script>
```

Comment:
We should **not** load vendor.js in this file. For one thing, it will not always be at the above URL (depending on build env), but it also contains _all_ the vendor libs. 

We will need to figure out how to either just include arcgis-rest-auth.js, or copy/paste the relevant code into this file. We can do the former by adding that file to the public tree, but I don't know if it will execute on it's own (or if we'll have to pull in arcgis-rest-request.js too even though I don't think it would be needed for this fn).

Reply:
good catch. i'll replace with a CDN link (for now anyway)